kumascript/macros/Compat.ejs

```diff
@@ -35,9 +35,10 @@ if (!query) {
   throw new Error("No first query argument or 'browser-compat' front-matter value passed");
 }
 var depth = $1 || 1;
```

Comment:
@sideshowbarker We might want to reduce the depth to 0 if we're showing multiple items.

Reply:
I thought the `depth` handling has been fixed in March or April by Vinyl or Daniel Jacobs. I'm not sure, but in doubt we just leave it as is for now.

I asked @fiji for review, and if you also give it a 👍 , we should be able to land this.